modules/dynamic-plugins/proc-package-third-party-dynamic-plugin.adoc

```diff
@@ -0,0 +1,165 @@
+[id="proc-package-third-party-dynamic-plugin_{context}"]
+= Packaging and publishing third-party plugins as dynamic plugins
+
+To package a third-party plugin as a dynamic plugin, you need to access to its source code and must export a plugin using the `@janus-idp/cli`. For more information, see xref:proc-export-third-party-plugins-rhdh_{context}[]. 
```

Comment:
"...and must export a plugin using `@janus-idp/cli'."

I think "the" is not necessary before '@janus-idp/cli'...

Reply:
+1, removed the article

Reply:
This paragraph has been removed.